packages/primereact/src/AutoCompleteWidget/AutoCompleteWidget.tsx

```diff
@@ -0,0 +1,91 @@
+import {
+  ariaDescribedByIds,
+  FormContextType,
+  getInputProps,
+  RJSFSchema,
+  StrictRJSFSchema,
+  WidgetProps,
+} from '@rjsf/utils';
+import { ChangeEvent, useState } from 'react';
```

Comment:
Let's try to keep react imports at the top:
```suggestion
import { ChangeEvent, useState } from 'react';
import {
  ariaDescribedByIds,
  FormContextType,
  getInputProps,
  RJSFSchema,
  StrictRJSFSchema,
  WidgetProps,
} from '@rjsf/utils';
```